[debug] [2023-01-04T21:06:45.283Z] ----------------------------------------------------------------------
[debug] [2023-01-04T21:06:45.285Z] Command:       /usr/local/Cellar/node/18.7.0/bin/node /usr/local/bin/firebase deploy --only functions
[debug] [2023-01-04T21:06:45.286Z] CLI Version:   11.19.0
[debug] [2023-01-04T21:06:45.286Z] Platform:      darwin
[debug] [2023-01-04T21:06:45.286Z] Node Version:  v18.7.0
[debug] [2023-01-04T21:06:45.289Z] Time:          Wed Jan 04 2023 22:06:45 GMT+0100 (GMT+01:00)
[debug] [2023-01-04T21:06:45.289Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-01-04T21:06:45.293Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2023-01-04T21:06:45.388Z] Object "" in "firebase.json" has unknown property: {"additionalProperty":"react-native"}
[debug] [2023-01-04T21:06:50.407Z] Error: Timed out.
    at Timeout._onTimeout (/usr/local/lib/node_modules/firebase-tools/lib/utils.js:253:49)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2023-01-04T21:08:15.653Z] ----------------------------------------------------------------------
[debug] [2023-01-04T21:08:15.657Z] Command:       /usr/local/Cellar/node/18.7.0/bin/node /usr/local/bin/firebase deploy --only functions:onWriteOrder
[debug] [2023-01-04T21:08:15.659Z] CLI Version:   11.19.0
[debug] [2023-01-04T21:08:15.659Z] Platform:      darwin
[debug] [2023-01-04T21:08:15.659Z] Node Version:  v18.7.0
[debug] [2023-01-04T21:08:15.661Z] Time:          Wed Jan 04 2023 22:08:15 GMT+0100 (GMT+01:00)
[debug] [2023-01-04T21:08:15.661Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-01-04T21:08:15.667Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2023-01-04T21:08:15.896Z] Object "" in "firebase.json" has unknown property: {"additionalProperty":"react-native"}
[debug] [2023-01-04T21:08:20.924Z] Error: Timed out.
    at Timeout._onTimeout (/usr/local/lib/node_modules/firebase-tools/lib/utils.js:253:49)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2023-01-04T21:08:58.490Z] ----------------------------------------------------------------------
[debug] [2023-01-04T21:08:58.493Z] Command:       /usr/local/Cellar/node/18.7.0/bin/node /usr/local/bin/firebase deploy --only functions:onWriteProcess
[debug] [2023-01-04T21:08:58.493Z] CLI Version:   11.19.0
[debug] [2023-01-04T21:08:58.493Z] Platform:      darwin
[debug] [2023-01-04T21:08:58.494Z] Node Version:  v18.7.0
[debug] [2023-01-04T21:08:58.495Z] Time:          Wed Jan 04 2023 22:08:58 GMT+0100 (GMT+01:00)
[debug] [2023-01-04T21:08:58.495Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-01-04T21:08:58.498Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2023-01-04T21:08:58.596Z] Object "" in "firebase.json" has unknown property: {"additionalProperty":"react-native"}
[debug] [2023-01-04T21:09:03.614Z] Error: Timed out.
    at Timeout._onTimeout (/usr/local/lib/node_modules/firebase-tools/lib/utils.js:253:49)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2023-01-04T21:10:16.102Z] ----------------------------------------------------------------------
[debug] [2023-01-04T21:10:16.105Z] Command:       /usr/local/Cellar/node/18.7.0/bin/node /usr/local/bin/firebase functions:log
[debug] [2023-01-04T21:10:16.105Z] CLI Version:   11.19.0
[debug] [2023-01-04T21:10:16.105Z] Platform:      darwin
[debug] [2023-01-04T21:10:16.106Z] Node Version:  v18.7.0
[debug] [2023-01-04T21:10:16.107Z] Time:          Wed Jan 04 2023 22:10:16 GMT+0100 (GMT+01:00)
[debug] [2023-01-04T21:10:16.107Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-01-04T21:10:16.111Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2023-01-04T21:10:16.208Z] Object "" in "firebase.json" has unknown property: {"additionalProperty":"react-native"}
[debug] [2023-01-04T21:10:21.238Z] Error: Timed out.
    at Timeout._onTimeout (/usr/local/lib/node_modules/firebase-tools/lib/utils.js:253:49)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
[error] 
[error] Error: An unexpected error has occurred.
